### PR TITLE
tap-setup-scripts for multi cluster

### PR DESCRIPTION
**Pull request by satya-dillikar**
tap-setup-scripts for multi cluster:
Add code for following items:

- TAP Scripts: TAP Profile "ootb_supply_chain_testing" on Iterate Cluster
- TAP Scripts: TAP Profile "ootb_supply_testing_scanning" on Build Cluster
- TAP Scripts: (Build+Run) & View clusters communication integration
- TAP Scripts: TAP-GUI & Metadata-store integration
- TAP Scripts: Build & Run cluster Service A/C to access from View Cluster

### CODE REVIEW

File: tap-setup-scripts/README_FIRST.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+## For Multi cluster
+
+update file user-input-values-mc.yaml with below params
+
+- the cluster name is a prefix of all 4 clusters
+- both build_cluster_arn & iterate_cluster_arn are required
+
+```
+cluster:
+  arch: multi
```
Comment (aaronmeza) [inline tap-setup-scripts/README_FIRST.md:13]: nit - _arch_ is an overloaded term. would prefer something like _topology_ for the key here and the variable used elsewhere.

File: tap-setup-scripts/src/inputs/tap-values-build.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#@ load("@ytt:data", "data")
+---
+profile: build
+ceip_policy_disclosed: true # Installation fails if this is set to 'false'
```
Comment (aaronmeza) [inline tap-setup-scripts/src/inputs/tap-values-build.yaml:4]: a question came up about ceip. do we create the configmap to enable it?

**Review by aaronmeza — APPROVED** [review]

**Review by aaronmeza — APPROVED** [review]

File: tap-setup-scripts/README_FIRST.md (hunk 1 above)
Comment (satya-dillikar) [reply tap-setup-scripts/README_FIRST.md:13]: sure, will do it in the next iteration.

File: tap-setup-scripts/src/inputs/tap-values-build.yaml (hunk 2 above)
Comment (satya-dillikar) [reply tap-setup-scripts/src/inputs/tap-values-build.yaml:4]: I am not sure about this.

**Pull request merged**